packages/garbo/src/diet.ts

```diff
@@ -1027,6 +1036,13 @@ export function consumeDiet(diet: Diet<Note>, name: DietName): void {
   );
   acquire(seasoningCount, $item`Special Seasoning`, MPA);
 
+  const aioliCount = sum(diet.entries, ({ menuItems, quantity }) =>
+    menuItems.some((menuItem) => menuItem.item === $item`mini kiwi aioli`)
+      ? quantity
+      : 0,
+  );
+  acquire(aioliCount, $item`mini kiwi aioli`, MPA * 1.5); // We reallly should set the price limit based on the adventure gain of the actual food
```

Comment:
We similarly shouldn't need this because it's not similar to seasoning, it's similar to forks